File: ruby/7_kyu/cw_getmiddle.rb
def get_middle(s)
  new_str = ""
  arr = s.split("")
  if arr.length % 2 != 0
    new_str = arr[arr.length/2]
  else
    new_str = arr[(arr.length/2)-1] + arr[(arr.length/2)] 
  end
 return new_str
end

#https://www.codewars.com/kata/56747fd5cb988479af000028

File: javascript/7_kyu/cw_getmiddle.js
function getMiddle(s) {
  return (s.length % 2 == 0) ? s.substr((s.length/2) - 1, 2) : s.substr((s.length/2), 1) 
}

//  https://www.codewars.com/kata/get-the-middle-character/javascript